### ISSUE TITLE
Use Faraday's own instrumentation middleware?

### ISSUE THREAD

**Issue by franzliedke** [mnemosyne-mon/mnemosyne-ruby#20]
Apparently Faraday added instrumentation support (via middleware) lately, or I missed this when originally implementing our own instrumentation layer in #11.

Using the official one (public API, basically) would feel a bit safer to me.

See https://lostisland.github.io/faraday/middleware/instrumentation.

**Comment by jgraichen** [comment]
There would still be the issue on how to enable for *all* clients included clients initialized in some libraries?

**Comment by franzliedke** [comment]
Yeah, and if it's really a new feature (not sure to be honest) it might not work for all versions...
